### DISCUSSION TITLE
Home Assistant Services

### DISCUSSION THREAD

**Discussion by lloydbayley** [Blackymas/NSPanel_HA_Blueprint#420] [Q&A] [answered]
G'day All!

I have looked/searched, and it has eluded me.
Is there somewhere in the documentation that describes some of the HA services and how/when to use them?

I get the obvious ones for notifications and page changes etc but would like to know the intricacies of ones like:

send_command_value
send_command_show/hide

Difference between:

send_command_printf
send_command_text_printf

etc.

Is there a page in the docs that I have missed?

Thanks!

**Comment by Blackymas** [comment, marked as answer]
it is not documented because we need it for our project.

but in the esphome file you can see what the services do

**Reply by lloydbayley** [reply]
I'm not sure I understand, you mean the documentation is not written yet?
If it makes things easier, I can understand German if you want to write in that.

Thank you!

**Reply by Blackymas** [reply]
Wir haben keine Dokumentation zu den meisten Services. Weil die für uns benötigt werden und es nicht angemacht war das es frei benutzt werden kann oder soll.

Die einzelnen Services werden in der esphome Datei erstellt - wenn du da reinschaust siehst du auch was dies Services machen oder kannst es bei der esphome Doku nachlesen

**Reply by lloydbayley** [reply]
Alles Klar, Schönen Dank für's antwort.

**Reply by Blackymas** [reply]
kein problem :)
